Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -1376,6 +1377,49 @@ bool RenderStyle::changeRequiresRecompositeLayer(const RenderStyle& other, Optio
     return false;
 }
 
+bool RenderStyle::scrollAnchoringSuppressionStyleDidChange(const RenderStyle* other) const
+{
+    if (!other)
+        return false;
+    if (m_nonInheritedData->boxData.ptr() != other->m_nonInheritedData->boxData.ptr()) {
+        if (m_nonInheritedData->boxData->width() != other->m_nonInheritedData->boxData->width()
+            || m_nonInheritedData->boxData->minWidth() != other->m_nonInheritedData->boxData->minWidth()
+            || m_nonInheritedData->boxData->maxWidth() != other->m_nonInheritedData->boxData->maxWidth()
+            || m_nonInheritedData->boxData->height() != other->m_nonInheritedData->boxData->height()
+            || m_nonInheritedData->boxData->minHeight() != other->m_nonInheritedData->boxData->minHeight()
+            || m_nonInheritedData->boxData->maxHeight() != other->m_nonInheritedData->boxData->maxHeight())
+            return true;
+    }
+    if (overflowAnchor() != other->overflowAnchor() && overflowAnchor() == OverflowAnchor::None)
+        return true;
+    if (m_nonInheritedFlags.position != other->m_nonInheritedFlags.position)
+        return true;
+    if (m_nonInheritedData->surroundData->margin != other->m_nonInheritedData->surroundData->margin)
+        return true;
+
+    if (m_nonInheritedData->surroundData->padding != other->m_nonInheritedData->surroundData->padding)
+        return true;
+
+    if (position() != PositionType::Static) {
+        if (m_nonInheritedData->surroundData->offset != other->m_nonInheritedData->surroundData->offset)
+            return true;
+    }
+
+    if (hasTransformRelatedProperty() != other->hasTransformRelatedProperty())
+        return true;
+
+    return false;
+}
+
+bool RenderStyle::absolutePositionStyleDidChange(const RenderStyle* other) const
+{
+    if (other && m_nonInheritedData.ptr() != other->m_nonInheritedData.ptr()) {
+        if (hasOutOfFlowPosition() != other->hasOutOfFlowPosition())
```

Comment:
This is affected by `position:fixed`  but the spec only refers to `position:absolute` changes. Again, a spec link would be useful.

Reply:
Added spec link but I think its referring to both position:fixed and position:absolute elements when it mentions absolutely positioned elements.